src/js/themes/base.d.ts

```diff
@@ -946,6 +950,30 @@ export interface ThemeType {
     colors?: GraphColorsType;
     extend?: ExtendType;
   };
+  pagination?: {
+    container?: {
+      extend: ExtendType;
+    };
+    controls?: {
+      align?: AlignContentType;
+      direction?:
```

Comment:
Can we have a `DirectionType` and re-use with Box?

Reply:
Done.